include/openssl/ssl.h

```diff
@@ -688,8 +685,7 @@ SSL_SESSION *(*SSL_CTX_sess_get_get_cb(SSL_CTX *ctx)) (struct ssl_st *ssl,
                                                        const unsigned char *data,
```

Comment:
',' is in column 81

Reply:
better fixed with a typedef on call_proc type.